suse_migration_services/prechecks/kernels.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2022
+#
+# SUSE Linux LLC.  All rights reserved.
+#
+# This file is part of suse-migration-services.
+#
+# suse-migration-services is free software: you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# suse-migration-services is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with suse-migration-services. If not, see <http://www.gnu.org/licenses/>
+#
+"""Checks the multiversion.kernels setting and the presence of multiple kernels
+
+Check to see if multiversion.kernels are enabled in /etc/zypp/zypp.conf and
+if there is more than one kernel currently installed. If detected, a warning is reported
+as multiversion.kernels needs to be disabled and old kernels cleaned up before
+a migration can be started."""
+
+import configparser
+import logging
+
+from suse_migration_services.defaults import Defaults
+from suse_migration_services.command import Command
+
+
+def multiversion_and_multiple_kernels():
+    """Check for the number of installed kernels"""
+    log = logging.getLogger(Defaults.get_migration_log_name())
+
+    config = configparser.ConfigParser()
+    config.read(Defaults.get_zypp_config_path())
+
+    kernel_multi_version_enabled = config.get('main', 'multiversion', fallback=None)
+
+    if 'kernel' in kernel_multi_version_enabled:
+        kernel_multi_version_settings = config.get('main', 'multiversion.kernels', fallback=None)
+
+        log.warning("The config option 'multiversion' in /etc/zypp/zypp.conf "
```

Comment:
is there an scenario where the `multiversion` and `multiversion.kernels` are set in a way that it is OK ? (in the context of DMS)
If so, should this log be a warning or info ?

Reply:
At this point we don not really know yet if this is a potential problem of not, as such this should be an "info" message, IMHO.

Reply:
In the other checks, IIRC the log messages are warnings because at that time it's known it _could_ be a problem, in this case
in this case, a check is going to be done, so inform the user about that check (instead of warning them) makes sense to me

Reply:
@jesusbv  I am not sure I am completely following. Are you agreeing with Robert's suggestion of making the first message "info"  (shown below)  and that is sufficient or are you suggesting we also log if the config setting `'multiversion.kernels` is correct?  

```
        log.info("The config option 'multiversion' in /etc/zypp/zypp.conf "
                    "includes the keyword 'kernel.' The current value is set "
                    "as \n'multiversion = %s'. \nChecking the config option "
                    "'multiversion.kernels' to see if multiple kernels are "
                    "also enabled", kernel_multi_version_enabled)
```

Reply:
Sorry, I was replying to [this](https://github.com/SUSE/suse-migration-services/pull/241#discussion_r855173983) comment,
trying to explain why this _should_ be a `log.info` message and why the other checks messages were warnings

I [suggested](https://github.com/SUSE/suse-migration-services/pull/241#discussion_r854966586) it should be a log info 
or thought that was clear

So, yes that change would make more sense to me